check_api/src/main/java/com/google/errorprone/scanner/HubSpotErrorHandler.java

```diff
@@ -60,6 +84,21 @@ public static void recordMissingCheck(String checkName) {
     flushErrors();
   }
 
+  private static void recordCheckLoadError(Throwable e) {
+    DATA.computeIfAbsent(INIT_ERROR, ignored -> ConcurrentHashMap.newKeySet())
+        .add(toInitErrorMessage(e));
+
+    flushErrors();
+  }
+
+  private static String toInitErrorMessage(Throwable e) {
+    if (Strings.isNullOrEmpty(e.getMessage())) {
+      return "Unknown error";
+    } else {
+      return e.getMessage();
```

Comment:
This will result in a message like: 
```
com.google.errorprone.bugpatterns.BugChecker: Provider com.hubspot.error.prone.checks.EnumWritableKeyCheck could not be instantiated
```
being stored in overwatch